Vagrantfile

```diff
@@ -124,7 +125,27 @@ Vagrant.configure("2") do |config|
       ]
     }
     ansible.extra_vars = {
-      "pxe_server_ip_addr" => ip_addr + '/' + ip_prefix
+      "pxe_server_ip_addr" => ip_addr_server + '/' + ip_prefix
     }
   end
 end
+
+Vagrant.configure("2") do |config|
+  config.vm.hostname = 'pxe-client'
+  config.vm.box = 'centos/7'
+  config.vm.define 'pxe-client'
+  config.ssh.username = 'root'
+
+  config.vm.network "private_network",
```

Comment:
Since this network definition occurs twice, it seems to cause vagrant-libvirt to define two NICs on the pxe-server0 network...

Reply:
Good point.